src/pybind/mgr/dashboard_v2/controllers/rbd_mirroring.py

```diff
@@ -0,0 +1,337 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+
+import json
+import re
+
+from functools import partial
+
+import rbd
+
+from dashboard_v2.controllers.dashboard import Dashboard
+from ..tools import ApiController, RESTController, ViewCache
+from .. import logger
+
+
+@ApiController('rbdmirror')
+class RbdMirror(RESTController):
+
+    def __init__(self):
+        self.pool_data = {}
+
+    def list(self):
+        _, data = self._get_daemons_and_pools()
+        content_data = self._get_content_data()
+
+        if isinstance(data, Exception):
+            logger.exception("Failed to get rbd-mirror daemons and pools")
+            raise type(data)(str(data))
+        else:
+            daemons = data.get('daemons', [])
+            pools = data.get('pools', {})
+
+        warnings = 0
+        errors = 0
+        for daemon in daemons:
+            if daemon['health_color'] == 'error':
+                errors += 1
+            elif daemon['health_color'] == 'warning':
+                warnings += 1
+        for _, pool in pools.items():
+            if pool['health_color'] == 'error':
+                errors += 1
+            elif pool['health_color'] == 'warning':
+                warnings += 1
+        return {'warnings': warnings, 'errors': errors, 'content_data': content_data}
```

Comment:
could  you replace this to: `{'warnings': warnings, 'errors': errors, 'data': content_data, 'status': status}`

Reply:
Yes

Reply:
I wonder if it would make sense to provide this summary and `content_data` in different API endpoints. Similar to the old dashboard.

Reply:
Regarding "content_data is quite big": I don't think it would change much because we would only move the big part to another endpoint but the part itself is still big. To exclude warnings and errors here wouldn't change much in my point of view.

But of course if the API structure should be similar to the old dashboard, I will adapt it.

What do you thing @tspmelo ?

Reply:
>But of course if the API structure should be similar to the old dashboard, I will adapt it.
I don't think we need to adapt the API 1:1, or i'm wrong?

Reply:
Did _v1 used the summary in the overall dashboard? If yes - and we want to do this too -, we should move this to a different endpoint.

Reply:
I agree with @sebastian-philipp that we should separate this result.

´warning´ and ´errors´ are only used on the toplevel api and need to be available to be consumed by the toplevel method defined in dashboard.py.

´status´ and ´content_data´ should continue to be available here.

Reply:
Okay